src/test-fixtures.js

```diff
@@ -279,11 +277,9 @@ export default {
         },
         duplicated_greeting_message() {
             return this.$gettext("Hello there!")
-        }
-    },
-    methods: {
-        async getGreetingMessageAnswer() {
-            return await Promise.resolve('General Kenobi!');
+        },
+        answer_message() {
+            return this.$gettext("General Kenobi! You are a bold one.")
```

Comment:
I don't understand that diff since it seems to me that you could leave the `getGreetingMessageAnswer` function there, just to check it isn't processed. Eventually with an additional comment.